scipy/stats/_qmc.py

```diff
@@ -306,6 +307,108 @@ def discrepancy(
                          f" {set(methods)!r}")
 
 
+def geometric_discrepancy(
+        sample: npt.ArrayLike,
+        method: Literal["mindist", "mst"] = "mindist",
+        metric: str = "euclidean") -> float:
+    """Discrepancy of a given sample based on its geometric properties.
+
+    Parameters
+    ----------
+    sample : array_like (n, d)
+        The sample to compute the discrepancy from.
+    method: str, optional
+        The method to use. One of ``mindist`` for minimum distance
+        or ``mst`` for minimum spanning tree.
+    metric: str or callable, optional
+        The distance metric to use. See the documentation
+        for ``scipy.spatial.distance.pdist``.
+
+    Returns
+    -------
+    discrepancy : float
+        Discrepancy (higher values correspond to greater sample uniformity).
+
+    Notes
+    -----
```

Comment:
We can do the same in `discrepancy` to link to this function. (GitHub does not allow me to make a suggestion "that far".)
```suggestion
    Returns
    -------
    discrepancy : float
        Discrepancy (higher values correspond to greater sample uniformity).

	See Also
    --------
    discrepancy

    Notes
    -----
```

Reply:
Done